notebooks/EOPF_Tutorials/EOPF_SAFE_ZARR_COMPARE_METADATA_SENTINEL2_L2A.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>Here it seems that you are printing all the metadata available from SAFE, without any filtering related to the ones available in Zarr. Please check.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EOPF-Sample-Service/eopf-sample-notebooks/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>